src/PKSim.Core/Snapshots/Mappers/ExtendedPropertyMapper.cs

```diff
@@ -42,16 +40,14 @@ private Task<ModelExtendedProperty> mapExtendedProperty<T>(SnapshotExtendedPrope
 
       public override Task<ModelExtendedProperty> MapToModel(SnapshotExtendedProperty snapshot)
       {
-         if (snapshot.Type == typeof(string))
-            return mapExtendedProperty(snapshot, option => option.ToString());
-
-         if (snapshot.Type == typeof(double))
+         if (double.TryParse(snapshot.Value.ToString(), out double _))
```

Comment:
why don't you define a parameter in the try parse that you can use if the parse works instead of parsing again?